### PR TITLE
Invalidate AccessTokens on password change

### PR DESCRIPTION
**Pull request by bajtos**
Invalidate all existing sessions (delete all access tokens) after user's password was changed.

This patch supersedes #2665

Connect to strongloop-internal/scrum-loopback#925

@loay @raymondfeng PTAL

### CODE REVIEW

**Comment by bajtos** [comment]
Many of the changes in the test file are whitespace-only, see https://github.com/strongloop/loopback/pull/3018/files?w=1

**Comment by loay** [comment]
@slnode test please

**Comment by loay** [comment]
@slnode test please

File: common/models/user.js [hunk 1]
```diff
@@ -648,6 +648,15 @@ module.exports = function(User) {
     err.statusCode = 422;
     throw err;
   };
+
+  User._invalidateAccessTokensOfUsers = function(userIds, cb) {
+    if (!userIds.length) return cb();
```
Comment (bajtos) [inline common/models/user.js]: Fixed - see 062c529

**Pull request merged**